pkg/reconciler/broker/controller.go

```diff
@@ -97,5 +109,13 @@ func NewController(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
+	triggerInformer.Informer().AddEventHandler(controller.HandleAll(
+		func(obj interface{}) {
+			if trigger, ok := obj.(*v1alpha1.Trigger); ok {
+				impl.EnqueueKey(types.NamespacedName{Namespace: trigger.Namespace, Name: trigger.Spec.Broker})
```

Comment:
I'd check that this properly unregisters deletions?  Maybe you can use the tracker?

Reply:
I did and it seemed to enqueue the broker properly. Is there other things that we'd get from tracker instead of enqueing this way?